test/trackler/problem_test.rb

```diff
@@ -4,25 +4,65 @@
 class ProblemTest < Minitest::Test
   def test_existing_problem
     problem = Trackler::Problem.new('hello-world', FIXTURE_PATH)
-
     assert problem.exists?, "hello world files not found in the metadata dir"
+  end
+
+  def test_slug
```

Comment:
I am definitely in the "Don't DRY tests" camp... 

But for anyone that is curious, the tables don't have to stop the tests... 

```ruby
  describe "Problem" do
  ARTIFACTS = {
    slug: 'hello-world',
    name: 'Hello World',
    blurb: 'Oh, hello.',
    description: "* hello\n* hello again\n",
    md_url: 'http://github.com/exercism/x-common/blob/master/exercises/hello-world/description.md',
  }
  before :each do
    @problem = Trackler::Problem.new('hello-world', FIXTURE_PATH)
  end

  it 'must exist' do
    assert @problem.exists?, "hello world files not found in the metadata dir"
  end

  ARTIFACTS.each do |meth, exp|
    it "#{meth}" do
      assert_equal exp, Trackler::Problem.new('hello-world', FIXTURE_PATH).send(meth)
    end
  end
end
```

Which will produce this result:

```text
Run options: --seed 18197

# Running:

.....................F.............................

Finished in 0.050927s, 1001.4278 runs/s, 1747.5897 assertions/s.

  1) Failure:
Problem#test_0006_md_url [/home/victor/Projects/Exercism/!exercism_org/trackler/test/trackler/problem_test.rb:22]:
--- expected
+++ actual
@@ -1 +1 @@
-"http://github.com/exercism/x-common/blob/master/exercises/hello-world/description.md"
+"https://github.com/exercism/x-common/blob/master/exercises/hello-world/description.md"


51 runs, 89 assertions, 1 failures, 0 errors, 0 skips
Coverage report generated for Run PID:  to /home/victor/Projects/Exercism/!exercism_org/trackler/coverage. 538 / 541 LOC (99.45%) covered.
```
Even with the failing test inserted.

As @Insti noted, it saves `row < 3*n lines` per test.  The only real advantage is the tabular view for comparison, reading, as humans.